license-tool/Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2020     Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+
+FROM docker.io/openjdk:15-jdk
+
+RUN curl -sL https://rpm.nodesource.com/setup_10.x | bash -
+
+RUN yum install -y git nodejs
+
+ARG MAVEN_VERSION=3.6.3
+ARG BASE_URL=https://apache.osuosl.org/maven/maven-3/${MAVEN_VERSION}/binaries
+
+RUN mkdir -p /usr/local/apache-maven /usr/local/apache-maven/ref \
+  && curl -fsSL -o /tmp/apache-maven.tar.gz ${BASE_URL}/apache-maven-${MAVEN_VERSION}-bin.tar.gz \
+  && tar -xzf /tmp/apache-maven.tar.gz -C /usr/local/apache-maven --strip-components=1 \
+  && rm -f /tmp/apache-maven.tar.gz \
+  && ln -s /usr/local/apache-maven/bin/mvn /usr/bin/mvn
+
+RUN npm install yarn -g
+
+RUN mkdir /workspace && cd /workspace && \
+    git clone https://github.com/eclipse/dash-licenses.git && \
```

Comment:
Should we depend on the particular commit here to avoid unexpected broker build?
I'm OK with depending on master at this point and if we face any issue - we'll be able to find working commit and depend on it instead

Reply:
Yes. I prefer to use the master branch until we don't have a release version. If we face any issue - we'll be able to find working commit and depend on it instead and create a new issue.